packages/boxel/addon/components/boxel/radio-input/usage.hbs

```diff
@@ -0,0 +1,99 @@
+      @name="example-radio-usage"
+      @checkedId={{this.checkedId}}
+      @disabled={{this.disabled}}
+      @hiddenBorder={{this.hiddenBorder}}
+      @hiddenCheckbox={{this.hiddenCheckbox}}
+      @layout={{this.layout}}
+      @spacing={{this.spacing}}
+      @containerWidth={{this.containerWidth}}
+      @gridColumnWidth={{this.gridColumnWidth}}
+    as |item|>
+      <item.component @onChange={{fn this.onChange item.data.id}}>
+        {{item.data.id}}
+      </item.component>
+    </Boxel::RadioInput>
+  </:example>
+
+  <:api as |Args|>
+    <Args.String
+      @name="groupDescription"
+      @description="Description for this group of radio buttons"
+      @value={{this.groupDescription}}
+      @onInput={{fn (mut this.groupDescription)}}
+      @optional={{true}}
+    />
+    <Args.Array
+      @name="items"
+      @description="Items which will be represented by radio buttons. Each should have a unique 'id' attribute."
+      @value={{this.items}}
+      @onInput={{fn (mut this.items)}}
+    />
+    <Args.String
+      @name="checkedId"
+      @description="The id of the currently checked/selected item."
+      @value={{this.checkedId}}
+      @onInput={{fn (mut this.checkedId)}}
+      @optional={{true}}
+    />
+    <Args.Bool
+      @name="disabled"
+      @description="Whether selection is disabled."
+      @defaultValue="false"
+      @value={{this.disabled}}
+      @onInput={{fn (mut this.disabled)}}
+      @optional={{true}}
+    />
+    <Args.Bool
+      @name="hiddenCheckbox"
+      @description="Visually hides the radio checkbox input."
+      @defaultValue="false"
+      @value={{this.hiddenCheckbox}}
+      @onInput={{fn (mut this.hiddenCheckbox)}}
+    />
+    <Args.Bool
+      @name="hiddenBorder"
+      @description="Visually hides the item border."
+      @defaultValue="false"
+      @value={{this.hiddenBorder}}
+      @onInput={{fn (mut this.hiddenBorder)}}
+    />
+    <Args.String
+      @name="spacing"
+      @description="Adjusts spacing level."
+      @defaultValue=""
+      @options={{array "default" "compact"}}
+      @onInput={{fn (mut this.spacing)}}
+      @value={{this.spacing}}
+    />
+    <Args.String
+      @name="layout"
+      @description="Layout of the radio buttons."
+      @defaultValue="flex"
+      @options={{array "default" "grid"}}
+      @onInput={{fn (mut this.layout)}}
+      @value={{this.layout}}
+    />
+    <Args.String
+      @name="gridColumnWidth"
+      @description="The `@layout` must be set to `grid`. Sets the minimum grid-column width. Items will expand to fill the available container width."
+      @defaultValue="100%"
+      @onInput={{fn (mut this.gridColumnWidth)}}
+      @value={{this.gridColumnWidth}}
+    />
+    <Args.String
+      @name="containerWidth"
+      @description="Width for the container. ie. `max-content`, `50%`, `400px`..."
+      @defaultValue="100%"
+      @onInput={{fn (mut this.containerWidth)}}
+      @value={{this.containerWidth}}
+    />
```

Comment:
This was the trickiest part, since I don't know how well it will work going forward with other examples. We want boxel components to have styling and layout out-of-the-box, so I set these attributes for it. If it doesn't work well for our needs, I'm open to reconsidering including these.